website/docs/conf_file.md

```diff
@@ -87,10 +87,10 @@ For example, if you want to see hidden files (the ones whose name starts with a
 If you almost always want those flags, you may define them as default in the configuration file file, with the `default_flags` setting.
 
 ```Hjson
-default_flags: gh
+default_flags: -gh
```

Comment:
I feel this could be clearer.

It kind of *implies* that you can specify a full string of flags inside the `"..."` exactly as you would on the CLI (e.g., `broot --flag1 --flag2=val -3` --> `default_flags: "--flag1 --flag2=val -3"`). But it would be really nice if that were stated explicitly, either in the prose description above, or via a more complex example.